zha/application/platforms/sensor/__init__.py

```diff
@@ -69,6 +69,7 @@
     CLUSTER_HANDLER_PRESSURE,
     CLUSTER_HANDLER_SMARTENERGY_METERING,
     CLUSTER_HANDLER_SOIL_MOISTURE,
+    CLUSTER_HANDLER_FLOW_MEASUREMENT,
```

Comment:
these appear to be otherwise alphabetized

Reply:
Thanks. Hopefully resolved in latest commit (plus other places I'd not stuck to alphabetical order). I also dropped _MEASUREMENT for consistency with other clusters/sensors (e.g. TemperatureMeasurement is called CLUSTER_HANDLER_TEMPERATURE).